ngraph/cmake/external_onnx.cmake

```diff
@@ -52,7 +52,15 @@ if(NOT ext_onnx_POPULATED)
     if(CMAKE_CROSSCOMPILING)
         set(ONNX_CUSTOM_PROTOC_EXECUTABLE ${SYSTEM_PROTOC})
     endif()
-    add_subdirectory(${ext_onnx_SOURCE_DIR} ${ext_onnx_BINARY_DIR} EXCLUDE_FROM_ALL)
+
+    function(AddOnnxSubDirectory)
```

Comment:
Maybe in future we can add a common method for components with the same problem.

Reply:
the method looks quite simple and I suppose we need this only in 1 :)